README.md

```diff
@@ -1 +1,266 @@
+
+*[Hello-Microverse]* is a HTML CSS project
+
+## 🛠 Built With <a name="built-with"></a>
+
+### Tech Stack <a name="tech-stack"></a>
+
+- HTML
+- CSS
+- Linters
+
+<!-- <details>
+  <summary>Client</summary>
+  <ul>
+    <li><a href="https://reactjs.org/">React.js</a></li>
+  </ul>
+</details>
+
+<details>
+  <summary>Server</summary>
+  <ul>
+    <li><a href="https://expressjs.com/">Express.js</a></li>
+  </ul>
+</details>
+
+<details>
+<summary>Database</summary>
+  <ul>
+    <li><a href="https://www.postgresql.org/">PostgreSQL</a></li>
+  </ul>
+</details> -->
+
+<!-- Features -->
+
+### Key Features <a name="key-features"></a>
+
+<!-- > Describe between 1-3 key features of the application. -->
+
+- *Understanding Git, GitHub and GitHub Workflow*
+- *Configuring Linters*
+- *Creating pull requests and merging*
+
+<!-- <p align="right">(<a href="#readme-top">back to top</a>)</p> -->
+
+<!-- LIVE DEMO -->
+
+## 🚀 Live Demo <a name="live-demo"></a>
+
+<!-- > Add a link to your deployed project.
+
+- [Live Demo Link](https://google.com) -->
+A demo link will be uploaded subsequently.
+
+<!-- <p align="right">(<a href="#readme-top">back to top</a>)</p> -->
+
+<!-- GETTING STARTED -->
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+<!-- > Describe how a new developer could make use of your project. -->
+
+To get a local copy up and running, follow these steps.
+
+### Prerequisites
+
+In order to run this project you need:
+
+- Visual Studio Code
+- Live server Extension
+<!--
+Example command:
+
+sh
+ gem install rails
+
+ -->
+
+### Setup
+
+Clone this repository to your desired folder:
+
+sh
+  cd my-folder
+  git clone https://github.com/ngalamac/Hello-Microverse.git
+
+
+### Install
+
+Install this project with:
+
+sh
+  cd my-folder
+  npm install
+
+
+
+### Usage
+
+To run the project, start the html file on live server
+
+### Run tests
+
+To run tests, run the following command:
+sh
+  npx hint .
+  npx stylelint "**/*.{css,scss}"
+
+
+### Deployment
+
+You can deploy this project using:
+
+sh
+  gh-pages
+  netlify
+  vercel
+
+
+<!-- <p align="right">(<a href="#readme-top">back to top</a>)</p> -->
+
+<!-- AUTHORS -->
+
+## 👥 Authors <a name="authors"></a>
+
+👤 *Ngala Mac*
+
+- GitHub: [@ngalamac](https://github.com/ngalamac/Hello-Microverse.git)
+<!-- - Twitter: [@twitterhandle](https://twitter.com/twitterhandle) -->
+
+<!-- 👤 *Author2*
+
+- GitHub: [@githubhandle](https://github.com/githubhandle)
+- Twitter: [@twitterhandle](https://twitter.com/twitterhandle)
+- LinkedIn: [LinkedIn](https://linkedin.com/in/linkedinhandle)
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p> -->
+
+<!-- FUTURE FEATURES -->
+
+<!-- ## 🔭 Future Features <a name="future-features"></a>
+
+> Describe 1 - 3 features you will add to the project.
+
+- [ ] *[new_feature_1]*
+- [ ] *[new_feature_2]*
+- [ ] *[new_feature_3]*
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p> -->
+
```

Comment:
- [ ] As requested by the previous reviewer, please uncomment this section and edit it. This is the future features section and it is required 😄